package.json

```
@@ -29,7 +29,11 @@
         "import": "./dist/esm-browser/index.js",
         "require": "./dist/commonjs-browser/index.js"
       },
-      "default": "./dist/esm-browser/index.js"
+      "default": "./dist/esm-browser/index.js",
+      "types": {
+        "module": "./types.d.mts",
```

```suggestion
        "import": "./types.d.mts",
```

I've not seen `module` as an export condition before - could you point to its origin?

---

I believe this is [a webpack thing](https://webpack.js.org/guides/package-exports/#target-environment-independent-packages), possibly adopted by other packaging tools as well?

---

Ah, cool. These are types tho, and the TS docs only point to node (where the spec comes from), and they don't have a `module` condition listed there: https://nodejs.org/api/packages.html#conditional-exports